pyhtools/attackers/web/email_scraper.py

```diff
@@ -0,0 +1,74 @@
+from bs4 import BeautifulSoup
+import requests, re, lxml
```

Comment:
do we need to add lxml package in our dependencies list?

Reply:
Probably; the scraper won't function without it.

Reply:
^^